scripts/ci/check_compliance.py

```diff
@@ -224,7 +224,7 @@ class DevicetreeBindingsCheck(ComplianceTest):
     """
     Checks if we are introducing any unwanted properties in Devicetree Bindings.
     """
-    name = "Devicetree Bindings"
+    name = "DevicetreeBindings"
```

Comment:
we should be able to pass `-m "Devicetree Bindings"`, but I agree it's better to not use spaces.

Reply:
Agreed. Anyway we are using `-e` now which simplifies a lot the invocation.